digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriDataImporter.scala

```diff
@@ -108,46 +117,93 @@ class TierekisteriDataImporter(vvhClient: VVHClient, oracleLinearAssetDao: Oracl
         println("\nFetch Lighting by Road Number " + roadNumber)
         val trLighting = tierekisteriLightingAsset.fetchActiveAssetData(lightingTR, roadNumber)
 
-        trLighting.foreach { tr => println("\nTR: roadNumber, roadPartNumber, start and end: " + tr.roadNumber + " " + tr.roadPartNumber + " " + tr.starMValue + " " + tr.endMValue) }
+        trLighting.foreach { tr => println("\nTR: roadNumber, roadPartNumber, start and end: " + tr.roadNumber + " " + tr.startRoadPartNumber + " " + tr.startAddressMValue + " " + tr.endAddressMValue) }
 
-        val newTRLighting = trLighting.groupBy(trLighting => (trLighting.roadNumber, trLighting.roadPartNumber, trLighting.starMValue, trLighting.endMValue)).map(_._2.head)
+        val newTRLighting = trLighting.groupBy(trLighting => (trLighting.roadNumber, trLighting.startRoadPartNumber, trLighting.startAddressMValue, trLighting.endAddressMValue)).map(_._2.head)
```

Comment:
What's this grouping function in this? This will pick some (random) TRLighting from all the objects that share these values?